contracts/Portfolio.sol

```diff
@@ -541,19 +482,14 @@ abstract contract PortfolioVirtual is Objective {
 
         // -=- Apply Fee Saving Method -=- //
         {
-            // Fees are saved by incrementing the fee growth accumulator.
-            bool saved = _feeSavingEffects(args.poolId, iteration);
-
             // If the fees are not saved,
```

Comment:
the comment should be changed, it only needs to talk about the reinvestment case.